docs/examples/bower.yml

```diff
@@ -2,4 +2,4 @@ excludes:
   scopes:
   - name: "devDependencies"
     reason: "BUILD_TOOL_OF"
-    comment: "These are dependencies only used for development. They are not distributed in the context of this product."
+    comment: "Scope with dependencies only used for development. Not included in released artifacts in the context of this project."
```

Comment:
optional: I personally dislike the redundancy introduced by 'Not included in released artifacts in the context of this project.' because not being distributed is what excludes are ALL ABOUT.

this can be considered out of scope of this change.

Reply:
Yes, you and I known that but persons who:
- stumble upon a .ort.yml file in a repository and do not know ORT will have a better chance of understanding what this file is about.
-  are non-technical readers of ORT scan report will instantly know how to interpret  excludes. Concept of excludes is not explained in the reports. Adding `Not included in released artifacts in the context of this project.` is a short-term workaround until In the long-term fix in the reports to explain excludes.